README.md

```diff
@@ -646,6 +883,320 @@ $util.qr($v1.put($b, true))
 #end
 ```
 
+## Typescript -> Event patterns
```

Comment:
Let put this in a collapsible section to save space? We need to later move these to dedicated markdown files for docusaurus. Readme is huge.